### DISCUSSION TITLE
How to preserver function events when deploying a function with the CLI?

### DISCUSSION THREAD

**Discussion by WuGGu** [appwrite/appwrite#7940] [Q&A] [answered]
### 👟 Reproduction steps

When I deploy a new version of my python 3.9 function with appwrite deploy functions. My custom Event to listen on creation of a document in a specific collection gets lost, and the Events field under Settings is empty again.
`databases.650f08cc543f36dba3f5.collections.650f5595c28eef540b1b.documents.*.create`

### 👍 Expected behavior

It sould instead keep the Event settings.
<img width="1188" alt="Bildschirmfoto 2024-04-08 um 15 34 00" src="https://github.com/appwrite/appwrite/assets/10403190/e096835d-d0f4-41bf-9282-a30d33ba23c0">

### 👎 Actual Behavior

It actually looses all the information and shows an empy card.
<img width="1179" alt="Bildschirmfoto 2024-04-08 um 15 32 53" src="https://github.com/appwrite/appwrite/assets/10403190/79f31861-d464-48b8-b624-baea33f7f236">

### 🎲 Appwrite version

Version 1.5.x

### 💻 Operating system

Linux

### 🧱 Your Environment

Rocky Linux, Hetzner 1.5.4

### 👀 Have you spent some time to check if this issue has been raised before?

- [X] I checked and didn't find similar issue

### 🏢 Have you read the Code of Conduct?

- [X] I have read the [Code of Conduct](https://github.com/appwrite/.github/blob/main/CODE_OF_CONDUCT.md)

**Comment by WuGGu** [comment]
Ok same for deployment with nodej 16.0 Runtime.

**Comment by stnguyen90** [comment, marked as answer]
@WuGGu, did you add the [events](https://appwrite.io/docs/tooling/command-line/deployment#function-options) in the appwrite.json?

**Reply by WuGGu** [reply]
Oh no! I'm sorry. Didn't saw that I have to configure this also in the appwrite.json. 😭 my bad.
